java-client/src/main/java/org/opensearch/client/transport/httpclient5/ResponseException.java

```diff
@@ -73,7 +73,7 @@ static String buildMessage(Response response) throws IOException {
 
         HttpEntity entity = response.getEntity();
         if (entity != null) {
-            if (entity.isRepeatable() == false) {
+            if (!entity.isRepeatable()) {
```

Comment:
Could you please revert this change? (this is the convention we use in OpenSearch for boolean checks, for better or worth)